server/routes/index.ts

```diff
@@ -574,6 +580,65 @@ export function defineRoutes(router: IRouter) {
     }
   );
 
+  router.get(
+    {
+      path: `${API_ENDPOINT_DASHBOARD_SIGNIN_OPTIONS}`,
+      validate: false,
+      options: {
+        authRequired: false,
+      },
+    },
+    async (
+      context,
+      request,
+      response
+    ): Promise<IOpenSearchDashboardsResponse<any | ResponseError>> => {
+      const client = context.security_plugin.esClient.asScoped(request);
+      let esResp;
```

Comment:
nit: `osResp` 😅

Reply:
I just followed the pattern I found in this file.